index.html

```diff
@@ -9222,6 +9225,256 @@ <h3 id="example-2">
             </dd>
           </dl>
         </section>
+        <section>
+          <h2>
+            Processing
+          </h2>
+          <p>
+            Dynamics compression can be implemented in a variety of ways. The
+            <a>DynamicsCompressorNode</a> implements a dynamics processor that
+            has the following characteristics:
+          </p>
+          <ul>
+            <li>Fixed look-ahead (this means that an
+            <a>DynamicsCompressorNode</a> adds a fixed latency to the signal
+            chain).
+            </li>
+            <li>Configurable attack speed, release speed, threshold, knee
+            hardness and ratio.
+            </li>
+            <li>Side-chaining is not supported
+            </li>
+            <li>The gain reduction is reported <em>via</em> the
+            <code>reduction</code> property on the
+            <a>DynamicsCompressorNode</a>
+            </li>
+            <li>The compression curve is has three parts:
+              <ul>
+                <li>The first part is the identify: \(f(x) = x\).
+                </li>
+                <li>The second part is the soft-knee portion, which MUST be a
+                monotonically increasing function.
+                </li>
+                <li>The third part is a linear function of equation \(f(x) =
+                \frac{1}{ratio} \cdot x \).
+                </li>
+              </ul>This curve MUST be continuously differentiable, and
+              corresponds to a target output level, based on the input level.
+            </li>
+          </ul>
+          <p>
+            Graphically, such a curve would look something like this:
+          </p>
+          <figure>
+            <img alt="Graphical representation of a compression curve" src=
+            "images/compression-curve.svg" style="width: 50%">
```

Comment:
In the diagram, you draw ratio as if it's the angle of the line after the knee.  It's not; it's the reciprocal of the slope.  Not exactly sure how to show that.